corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -827,6 +827,8 @@ hqDefine("cloudcare/js/form_entry/entrycontrols_full", function () {
         self._calendarInstance = $.calendars.instance('ethiopian', calendarLanguage);
         if (calendarLanguage === 'en') {
             $.calendarsPicker.setDefaults($.calendarsPicker.regionalOptions['']);
+        } else {
+            $.calendarsPicker.setDefaults($.calendarsPicker.regionalOptions[calendarLanguage]);
```

Comment:
`regionalOptions` doesn't let you pass in `en`? That's an odd decision.

Reply:
everything about this widget made me feel the same way, but it was the only one I could find with reasonably good ethiopian date support.

Reply:
haha, fair